calcfetch

```diff
@@ -96,16 +92,16 @@ memstat="${mem_used}/${mem_full} MB"
 
 ## DEFINE COLORS
 
-bold="$(tput bold)"
-black="$(tput setaf 0)"
-red="$(tput setaf 1)"
-green="$(tput setaf 2)"
-yellow="$(tput setaf 3)"
-blue="$(tput setaf 4)"
-magenta="$(tput setaf 5)"
-cyan="$(tput setaf 6)"
-white="$(tput setaf 7)"
-reset="$(tput sgr0)"
```

Comment:
tput sux lol

Reply:
damn tput do be tputting tho :flushed:

Reply:
i mean, basename might have its uses if multiple args are provided, not sure, never used it.

tput is

 - non-standard (basename should be installed, its in posix)
 - literally useless in this context, it could be replaced with what I did below with literally no (visual) difference, and a lot less overhead.